daemon/image_delete.go

```diff
@@ -14,26 +12,20 @@ import (
 	"github.com/docker/docker/utils"
 )
 
-func (daemon *Daemon) ImageDelete(job *engine.Job) error {
-	if n := len(job.Args); n != 1 {
-		return fmt.Errorf("Usage: %s IMAGE", job.Name)
-	}
-
+// FIXME: remove ImageDelete's dependency on Daemon, then move to graph/
```

Comment:
I moved this reminder here from daemon/daemon.go (I'm pretty sure I'm not able to do this refactor, in case just close this pr)